Description: Adapt to GHC 7.4 filename handling
 GHC 7.4 already deals with unicode filenames, no need to encode/decode anymore.
Author: Iustin Pop <iustin@debian.org>
Forwarded: no
Last-Update: 2012-05-19
--- a/shelltest.hs
+++ b/shelltest.hs
@@ -467,8 +467,4 @@
 -- (cf http://www.dwheeler.com/essays/fixing-unix-linux-filenames.html)
 -- and elsewhere we leave it unchanged.
 toPlatformString :: String -> PlatformString
-toPlatformString = case os of
-                     "unix" -> UTF8.encodeString
-                     "linux" -> UTF8.encodeString
-                     "darwin" -> UTF8.encodeString
-                     _ -> id
+toPlatformString = id
